libsupport/src/Strings.cpp

```diff
@@ -41,3 +41,41 @@ katana::TrimSuffix(const std::string& s, const std::string& suffix) {
   }
   return s;
 }
+
+std::string
+katana::FromBase64(std::string input) {
+  using namespace boost::archive::iterators;
```

Comment:
this namespace is only used once in the typedef below, consider just writing it out there.